prince-of-versions/src/main/java/co/infinum/princeofversions/PrinceOfVersionsDefaultStorage.java

```diff
@@ -0,0 +1,29 @@
+package co.infinum.princeofversions;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+/**
+ * Implementation of {@link Storage} which stores data in default {@link SharedPreferences}.
+ */
+public class PrinceOfVersionsDefaultStorage implements Storage {
+
+    private static final String KEY = "PrinceOfVersions_LastNotifiedUpdate";
+
+    private SharedPreferences sp;
+
+    public PrinceOfVersionsDefaultStorage(Context context) {
+        sp = PreferenceManager.getDefaultSharedPreferences(context);
```

Comment:
That's good suggestion, but then we also need to implement some migration because from this default to new file.

Reply:
I understand, but maybe that would be wise to do sooner rather than later. I don't think it's a good idea to expose our data because it could be deleted.